src/lib/browse/LayerControls.svelte

```diff
@@ -69,6 +70,7 @@
   </CollapsibleCard>
   <CollapsibleCard label="Other">
```

Comment:
"Statistics"? "Counts"?

Reply:
I think the problem is that that there are things that fit those descriptions in other categories. Maybe merge with census and call em statistics?

Incidentally, should criticals really be over on the right now that I'm thinking about these layers categories? It seems like there's a delineation between left and right: left is ATE data which we might want to interact with directly, right is precomputed context data which is static as far as ATIP is concerned?

Reply:
I agree the left/right differences are very weird right now. @stuartlynn described some nice ideas about rethinking the UI more generally here. First a user would load data and search through layers to enable. (And that bit could do a better job maybe of preventing you from enabling multiple boundary layers at once, or a boundary and a census layer.)  Then we have a single sidebar/control that only has a legend and any filters/controls for those layers. The user could go back and opt into different layers.

If we went that route, it'd also be a potentially nice time to try and encode the current settings for enabled layers (and their settings) in some way that could be shared as a permalink, expressed in the URL, etc.

Reply:
Going to try and have a prototype for this approatch early next week so we can discuss it's merits and if it feels like an improvment over the current approatch